### PR TITLE
depends: update README.md with working Android targets and API levels

### PR DESCRIPTION
**Pull request by icota**
Per @Sjors comments in https://github.com/bitcoin/bitcoin/pull/16110#pullrequestreview-310821810

### CODE REVIEW

File: depends/README.md [hunk 1]
```diff
@@ -30,11 +30,17 @@ Common `host-platform-triplets` for cross compilation are:
 - `aarch64-linux-gnu` for Linux ARM 64 bit
 - `riscv32-linux-gnu` for Linux RISC-V 32 bit
 - `riscv64-linux-gnu` for Linux RISC-V 64 bit
+- `armv7a-linux-android` for Android ARM 32 bit
 - `aarch64-linux-android` for Android ARM 64 bit
+- `i686-linux-android` for Android x86 32 bit
+- `x86_64-linux-android` for Android x86 64 bit
 
 The paths are automatically configured and no other options are needed unless targeting Android.
 Before proceeding with an Android build one needs to get the [Android SDK](https://developer.android.com/studio) and use the "SDK Manager" tool to download the NDK and one or more "Platform packages" (these are Android versions and have a corresponding API level).
```
Comment (laanwj) [inline depends/README.md]: Maybe make this information for Android a separate section

Comment (icota) [reply depends/README.md]: Sure thing

**Comment by emilengler** [comment]
Wouldn't it be better to create a file for this in the `doc` folder and link to it in the README?

**Comment by icota** [comment]
I don't think so. This is specific to depends and other platform cross-compilation instructions are in the same file.

**Comment by Sjors** [comment]
It's fine. It's not that long yet, and if you search the repo for Android you'll find it :-)

ACK 3fe1aba

**Comment by cryptozeny** [comment]
thanks alot!

**Comment by Sjors** [comment]
Future note: please don't tag Github usernames in commit messages; it results in spam when altcoins backport it. My own fault for missing that :-)

**Comment by icota** [comment]
You didn't miss anything. :smile: 

I read `CONTRIBUTING` like a good boy so I didn't include it in any commit messages. Merge commit picked it up from the first comment though. Should we avoid that?

**Comment by Sjors** [comment]
No, I think the merge script should take care of that. I thought it stripped the `@`. @MarcoFalke?

**Comment by laanwj** [comment]
No, it doesn't strip them—as this might corrupt, say, mail addresses, github's parsing is pretty ambigious and heuristic-based!
it warns when there are any before merging, though

**Pull request merged**